src/sentry/api/endpoints/organization_releases.py

```diff
@@ -269,7 +267,6 @@ def get(self, request, organization):
             **paginator_kwargs
         )
 
-    @transaction_start("OrganizationReleasesEndpoint.post")
```

Comment:
Why are these being removed?

Reply:
These getting removed is the main motivation behind this PR, currently when these endpoints are being hit from the frontend these are transactions with transactions.
- Which starts a new transaction in the automatically instrumented one, losing a bunch of data in the automatic one and causing you to look for this manual one.
- These manual transactions don't pickup any of the automatic tags (eg. status code etc)

With the traces_sampler these will get transactions even when hit externally, and get all of their automatic tags.